web/src/components/TaskDetails.vue

```diff
@@ -0,0 +1,158 @@
+<!--
+Copyright 2022 Google Inc. All rights reserved.
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+<template>
+  <div>
+    <v-card-title>
+      {{ taskDetails.name }}
+      <v-tooltip top>
+        Download Task output
+        <template v-slot:activator="{ on, attrs }">
+          <v-btn icon v-on="on" v-bind="attrs" @click="getTaskOutput(taskDetails.id)">
+            <v-icon> mdi-file-download-outline </v-icon>
+          </v-btn>
+        </template>
+      </v-tooltip>
+    </v-card-title>
+    <v-alert v-if="taskDetails.successful === true" type="success" border="bottom" colored-border>
```

Comment:
if `taskDetails.successful` is a boolean, it is enough to do `v-if="taskDetails.successful`